From foo@baz Fri Nov  7 12:50:13 PST 2008
Date: Fri, 07 Nov 2008 12:50:13 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: benet: build is broken unless CONFIG_NETPOLL is enabled

So disable the driver in that build situation and document
that this needs to be fixed.

Cc: Subbu Seetharaman <subbus@serverengines.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/benet/Kconfig |    2 +-
 drivers/staging/benet/TODO    |    1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/staging/benet/Kconfig
+++ b/drivers/staging/benet/Kconfig
@@ -1,6 +1,6 @@
 config BENET
 	tristate "ServerEngines 10Gb NIC - BladeEngine"
-	depends on PCI && INET
+	depends on PCI && INET && NETPOLL
 	select INET_LRO
 	help
 	  This driver implements the NIC functionality for ServerEngines
--- a/drivers/staging/benet/TODO
+++ b/drivers/staging/benet/TODO
@@ -3,6 +3,7 @@ TODO:
 	- fix minor checkpatch.pl issues
 	- remove wrappers around common iowrite functions
 	- full netdev audit of common problems/issues
+	- fix build errors when CONFIG_NETPOLL is not enabled
 
 Please send all patches and questions to Subbu Seetharaman
 <subbus@serverengines.com> and Greg Kroah-Hartman <greg@kroah.com>
